docs/editing.md

```diff
@@ -174,6 +184,20 @@ This doesn’t mean that you’re the editor, just that you’ve been suggested
 
 ![Editorial flow](images/JOSS-flowchart.png)
 
+## Expectations on JOSS editors
+
+### Responding to editorial assignments
+
+As documented above, papers will be assigned to you by one of the EiCs. We ask that editors respond in a timely fashion (< 72 hours) to invites to edit a new submission.
+
+### Continued attention to assigned submissions
+
+As an editor, part of your job is to ensure that submissions you're responsible for are progressing smoothly through the editorial process. This means that once or twice per week you should check your GitHub notifications and/or your editorial dashboard (e.g. `http://joss.theoj.org/dashboard/arfon`) for updates to the papers you are handling.
```

Comment:
can we make the dashboard URL more generic somehow?

Reply:
In the docs here or more generally for editors?

Reply:
This one specifically. Something other than http://joss.theoj.org/dashboard/arfon would be good